### ISSUE TITLE
Use an icon for the `Move to Trash` button in the post editor instead.

### ISSUE THREAD

**Issue by aurooba** [WordPress/gutenberg#50294]
## What problem does this address?

This came out of [the discussion on Twitter](https://twitter.com/aurooba/status/1653804824524361729) that was triggered by the recent PR that was merged to move the `Switch to Draft` button into the Posts settings area in the Post Editor (#50217). Now that #50219 is also merged, can we/should we explore turning the `Move to Trash` button into an icon instead?

## What is your proposed solution?

<img width="284" alt="a small switch to draft button and a trash icon underneath the author dropdown in the WordPress post settings" src="https://user-images.githubusercontent.com/6925260/236030651-b91e6a2a-0ace-4eae-a725-55ea20e655cc.png">

I was imagining something like the mockup above, although I'm sure it needs refinement. Although _ideally_ it would be nice to have a variation of the `trash` icon with the lid slightly open to indicate that you'd be _moving_ something to the trash.

Since there's a move to change many of the post actions in the right hand of the Editor toolbar to icons as well (#50260, #50218), it makes sense to evaluate other post actions and see if they are candidates for icons as their default appearance as well.

**Comment by afercia** [comment]
Thanks for your feedback and interesting suggestion.
From an accessibility perspective the editor should strive to rather reduce the amount of icons and use labels with visible text. Proliferation of icons in the editor is a hug problem.

Icons usage should be reduced to a limited set of very well recognizable icons. Whle I do agree the 'bin' icon is well recognizable, I'd also note that it is generally used for the concept of 'delete', as in _permanently_ delete. Move to trash is a pretty different logical concept and action so I'm not sure the bin icon would ever be a good one for this specific case.

Also noting https://github.com/WordPress/gutenberg/issues/65023 just restored the big red 'Move to trash' button that was previously hidden behind the Post Summary panel three dot menu:

**Comment by Mamaduka** [comment]
Considering the a11y feedback and the fact that the "Switch to Draft" button is no longer in the same space, I'm going to close the issue.
